apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -342,6 +342,15 @@ export const ScreeningDecisionDialog = ({
           skill,
         );
 
+  const experienceAttachedToSkill = getExperienceSkills(
+    unpackMaybes(parsedSnapshot?.experiences),
+    skill,
+  )?.some((experience) =>
+    experience?.skills?.find(
+      (experienceSkill) => experienceSkill.id === poolSkill?.skill?.id,
+    ),
+  );
```

Comment:
This feels redundant - doesn't getExperienceSkills already do this?

```suggestion
  ).length > 0;
```

Reply:
Unfortunately not redundant. The function `getExperienceSkills` "Filters an array of experiences to get only the ones associated with a specific skill" but we still need to check for equality between the applicant's skill and the pool's skill.